src/ntlm/test.rs

```diff
@@ -1,677 +1,677 @@
-use lazy_static::lazy_static;
```

Comment:
You need to re-introduce these tests, I think

Reply:
updated